cedar-policy-symcc/tests/utils/mod.rs

```diff
@@ -100,39 +100,124 @@ impl<'a> Environments<'a> {
             symenv,
         }
     }
+
+    /// Gets all possible request environments from a schema.
+    /// Adapted from CedarSymEvalCLI.
+    pub fn get_all_from_schema(schema: &'a Schema) -> Vec<Self> {
```

Comment:
Can we use [Schema::request_envs()](https://docs.rs/cedar-policy/latest/cedar_policy/struct.Schema.html#method.request_envs) to simplify this?